resilience4j-spring/build.gradle

```diff
@@ -1,10 +1,11 @@
 dependencies {
-    compile ( libraries.aspectj )
-    compileOnly ( libraries.hibernate_validator )
-    compileOnly ( libraries.spring_4_core, libraries.spring_4_context )
+    compile(libraries.aspectj)
+    compileOnly(libraries.hibernate_validator)
+    compileOnly(libraries.spring_4_core, libraries.spring_4_context)
     compile project(':resilience4j-annotations')
     compile project(':resilience4j-consumer')
     compile project(':resilience4j-circuitbreaker')
+    compile project(':resilience4j-reactor')
```

Comment:
I'm not sure about the compile dependency to resilience4j-reactor.
resilience4j-spring is used by pure Spring applications and resilience4j-spring-boot1 and resilience4j-spring-boot2.
This would introduce Spring Reactor as a compile dependency to all applications using resilience4j-spring, even if they don't want to use it.
Maybe we can change it to `compilyOnly`